### DISCUSSION TITLE
Has anyone ever encountered an error like this? Previously it was running fine.

### DISCUSSION THREAD

**Discussion by ahmadfauzanarif** [serenity-is/Serenity#6609] [Help]
![image](https://user-images.githubusercontent.com/71299117/206352861-3fd1fd61-a76e-48b1-baf9-a2c7c9853e21.png)

**Comment by VictorTomaili** [comment]
What is changed from previous running state?

The exception says when your application started, and while row instance creation, you got stack overflow. The exception seems not directly related to serenity, and mostly occurs when 2 object needs each other. For example, Class A requires Class B to initialization. And Class B requires Class A to initialization. This means every time you want to create a Class A, it tries to create B and B tries to create A and so on until crash.
